.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] "
+labels: bug
+assignees: gitbuda, antonio2368
+
+---
+
+
+**Memgraph version**
+Which version did you use?
+
+**Environment**
+Some information about environment you are using Memgraph: operating system,
+how do you connect, with or without docker, which driver etc.
```

Comment:
This is new to us, it wasn't part of MAGE's issue template.

Reply:
Can we be more precise here? State explicitly all the information we require in a list?